Python/bytecodes.c

```diff
@@ -0,0 +1,3983 @@
+#include "Python.h"
+
+#include "opcode.h"
+#include "pycore_atomic.h"
+#include "pycore_frame.h"
+
+void _PyFloat_ExactDealloc(PyObject *);
+void _PyUnicode_ExactDealloc(PyObject *);
+
+#define SET_TOP(v)        (stack_pointer[-1] = (v))
+#define GETLOCAL(i)     (frame->localsplus[i])
+
+#define inst(name, stack_effect) case name:
+#define family(name) static int family_##name
+
+static void
+dummy_func(
+    PyThreadState *tstate,
+    _PyInterpreterFrame *frame,
+    unsigned char opcode,
+    unsigned int oparg,
+    _Py_atomic_int * const eval_breaker,
+    _PyCFrame cframe,
+    PyObject *names,
+    PyObject *consts,
+    _Py_CODEUNIT *next_instr,
+    PyObject **stack_pointer
+)
+{
+    switch (opcode) {
+
+        /* BEWARE!
+           It is essential that any operation that fails must goto error
+           and that all operation that succeed call DISPATCH() ! */
+
+// BEGIN BYTECODES //
+        // stack effect: ( -- )
+        inst(NOP) {
+        }
+
+        // stack effect: ( -- )
+        inst(RESUME) {
+            _PyCode_Warmup(frame->f_code);
+            GO_TO_INSTRUCTION(RESUME_QUICK);
+        }
+
+        // stack effect: ( -- )
+        inst(RESUME_QUICK) {
+            PREDICTED(RESUME_QUICK);
```

Comment:
Autogenerating these `PREDICTED` targets (and *maybe* even the `PREDICT` part) would be a good future project too.